DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -10584,7 +10628,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [ \"$CONFIGURATION\" != \"Debug\" ] || [ \"$ENABLE_PREVIEWS\" = \"YES\" ]; then exit 0; fi\n${BUILT_PRODUCTS_DIR}/SwiftLintTool\n";
+			shellScript = "if [ \"$CONFIGURATION\" != \"Debug\" ] || [ \"$ENABLE_PREVIEWS\" = \"YES\" ]; then exit 0; fi\n#${BUILT_PRODUCTS_DIR}/SwiftLintTool\n";
```

Comment:
This was causing me some compilation issues - I'll make sure to uncomment it before we merge.